include/ftl/atomic_counter.h

```diff
@@ -48,10 +48,13 @@ class AtomicCounter {
  * If a user tries to have more fibers wait on a counter, the fiber will not be tracked,
  * which will cause WaitForCounter() to infinitely sleep. This will probably cause a hang
  */
-#define NUM_WAITING_FIBER_SLOTS 4
+#ifndef NUM_WAITING_FIBER_SLOTS
```

Comment:
We should prefix all the macros as they are not namespaced :/.

Reply:
Ah good call. I'll fix them all in a new PR